include/vnum.h

```diff
@@ -31,6 +31,7 @@
  */
 
 /* from libvarnish/vnum.c */
+#define VNUM_EPSILON 1e-9
```

Comment:
Maybe something like `#define VNUM_EPSILON (1e-9d)` ?

Reply:
there you go